.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json
+{
+	"name": "pallets/flask",
+	"image": "mcr.microsoft.com/vscode/devcontainers/python:3",
+
+	// Configure tool-specific properties.
+	"customizations": {
```

Comment:
The [python image](https://github.com/devcontainers/images/blob/main/src/python/.devcontainer/devcontainer.json#L14) adds a python Feature, which contains this info. Hence, can we remove `customizations` to avoid duplication?

https://github.com/devcontainers/features/blob/main/src/python/devcontainer-feature.json#L56-L75

Reply:
This is what's being used to activate the `venv` in each terminal, so I can't fully remove it - I can clean it up though.